develop/addons/components/genericsetup.rst

```diff
@@ -717,203 +742,1393 @@ Example:
+      <property name="image_types">
+        <element value="Image"/>
+        <element value="News Item"/>
+      </property>
+      <property name="folder_types">
+        <element value="Image"/>
+      </property>
+      <property name="album_batch_size">30</property>
+      <property name="album_image_scale">thumb</property>
+      <property name="single_image_scale">preview</property>
+    </atcttool>
+
+
+portal_placeful_workflow
+------------------------
+
+This handles the ``portal_placeful_workflow.xml`` file and the ``portal_placeful_workflow`` directory.
+
+This install or configures a placeful workflow.
+For this to work, you must install Workflow Policy Support (CMFPlacefulWorkflow) in the add-ons.
+This add-on is included in standard Plone, but not activated.
+
+Standard ``portal_placeful_workflow.xml`` from ``Products.CMFPlacefulWorkflow``:
+
+.. code-block:: xml
+
+    <?xml version="1.0"?>
+    <object name="portal_placeful_workflow" meta_type="Placeful Workflow Tool">
+      <property name="title"></property>
+      <property name="max_chain_length" type="int">1</property>
+      <object name="intranet" meta_type="WorkflowPolicy"/>
+      <object name="old-plone" meta_type="WorkflowPolicy"/>
+      <object name="one-state" meta_type="WorkflowPolicy"/>
+      <object name="simple-publication" meta_type="WorkflowPolicy"/>
+    </object>
+
+Standard ``portal_placeful_workflow/simple-publication.xml`` from ``Products.CMFPlacefulWorkflow``:
+
+.. code-block:: xml
+
+    <?xml version="1.0"?>
+    <object name="simple-publication" meta_type="WorkflowPolicy">
+      <property name="title">Simple publication</property>
+      <bindings>
+        <default>
+          <bound-workflow workflow_id="simple_publication_workflow"/>
+        </default>
+        <type default_chain="true" type_id="Document"/>
+        <type default_chain="true" type_id="Event"/>
+        <type type_id="File">
+        </type>
+        <type default_chain="true" type_id="Folder"/>
+        <type type_id="Image">
+        </type>
+        <type default_chain="true" type_id="Link"/>
+        <type default_chain="true" type_id="News Item"/>
+        <type default_chain="true" type_id="Collection"/>
+      </bindings>
+    </object>
+
+Uninstall example:
+
+.. code-block:: xml
+
+    <?xml version="1.0"?>
+    <object name="portal_placeful_workflow" meta_type="Placeful Workflow Tool">
+      <object name="old-plone" meta_type="WorkflowPolicy" remove="true" />
+    </object>
+
+The import handler is in ``Products.CMFPlacefulWorkflow.exportimport.importWorkflowPolicies``.
+
+
+portlets.xml
+------------
+
+Code is in ``plone.app.portlets.exportimport.portlets``.
+
+.. automodule:: plone.app.portlets.exportimport.portlets
+
+
+propertiestool.xml
+------------------
+
+.. note::
+
+    This is deprecated in Plone 5.
+    Most properties are now handled in the configuration registry and can be configured in ``registry.xml``.
+
```

Comment:
Same here, since we have this already on a couple of places, maybe we should rewrite the doc in a way, which is mentioning all the depredations once like 'changes in versions' or something, otherwise we should use .. deprecated:: version